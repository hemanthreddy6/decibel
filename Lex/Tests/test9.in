audio_1 = audio_1 | audio_2;

function exp <- (length:float, time: float) {
	return 2^(-length*time);
}

audio_1*exp(10, 1.2); 

audio_2 >> 2.5;
audio_3 = ~audio_1[40:];