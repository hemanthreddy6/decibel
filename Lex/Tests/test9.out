IDENTIFIER
=
IDENTIFIER
|
IDENTIFIER
;
FUNCTION
IDENTIFIER
<-
(
IDENTIFIER
:
FLOAT
,
IDENTIFIER
:
FLOAT
)
{
RETURN
INT_LITERAL
^
(
-
IDENTIFIER
*
IDENTIFIER
)
;
}
IDENTIFIER
*
IDENTIFIER
(
INT_LITERAL
,
FLOAT_LITERAL
)
;
IDENTIFIER
>>
FLOAT_LITERAL
;
IDENTIFIER
=
~
IDENTIFIER
[
INT_LITERAL
:
]
;
